lib/modules/manager/gomod/update.ts

```diff
@@ -36,11 +35,14 @@ export function updateDependency({
     const lineToChange = lines[upgrade.managerData.lineNumber];
     logger.trace({ upgrade, lineToChange }, 'go.mod current line');
     if (
-      !lineToChange.includes(depNameNoVersion) &&
+      !lineToChange.includes(fromPackageNameNoVersion) &&
```

Comment:
Should we also check `toPackageNameNoVersion` here?

Reply:
It depends if this code gets executed the "second" time (next run) after a replacement. If so it looks like you need to check both